source/mainnet/smart-contracts/guides/no-std.rst

```diff
@@ -26,11 +32,45 @@ This feature is enabled by default.
 
 To disable it, one must simply disable default features for the
 ``concordium-std`` in the dependencies of your module.
+You can update your `Cargo.toml` file by using:
 
 .. code-block:: rust
 
    [dependencies]
-   concordium-std = { version: "=0.2", default-features = false }
+   concordium-std = { version = "6.0", default-features = false }
+
+.. note::
+
+   To compile your smart contracts, a memory `allocator <https://docs.rs/concordium-std/6.0.0/concordium_std/#use-a-custom-allocator>`_ is used.
+   ``concordium-std`` version ``<6.0.0`` hard-coded the use of the `wee_alloc <https://docs.rs/wee_alloc/>`_ allocator.
+   In ``concordium-std`` version ``>=6.0.0``, ``wee_alloc`` is a feature and needs to be explicitly enabled.
+   When ``std`` feature is enabled, the allocator provided by the Rust standard library is used
+   by default but when the ``wee_alloc`` feature is enabled in addition, `wee_alloc <https://docs.rs/wee_alloc/>`_ is used instead.
+
+   You can enable the ``std`` feature and the ``wee_alloc`` feature in ``concordium-std`` version
+   ``>=6.0.0`` by using:
+
+   .. code-block:: rust
+
+      [features]
+      default = ["std", "wee_alloc"]
+      std = ["concordium-std/std"]
+      wee_alloc = ["concordium-std/wee_alloc"]
```

Comment:
```suggestion
   You can enable the ``std`` feature and the ``wee_alloc`` feature in ``concordium-std`` version
   ``>=6.0.0`` by using:

   .. code-block:: rust

      [features]
      default = ["std", "wee_alloc"]
      std = ["concordium-std/std"]
      wee_alloc = ["concordium-std/wee_alloc"]
```

I still think my comment about the two options was not addressed. What I would like to see is to say something like this

```suggestion
   You can enable the ``std`` feature and the ``wee_alloc`` feature in ``concordium-std`` version
   ``>=6.0.0`` by using

   .. code-block:: rust

      [dependencies]
      concordium-std = {version = "6.0", features = ["wee_alloc"]}
      
   Alternatively, if you want to test with and without ``wee_alloc`` enabled add a ``wee_alloc`` feature to the smart contract crate like so
   
      .. code-block:: rust
      [features]
      default = ["std", "wee_alloc"]
      std = ["concordium-std/std"]
      wee_alloc = ["concordium-std/wee_alloc"]
```

Does this make any sense?